addons/l10n_ch/models/res_bank.py

```diff
@@ -117,12 +114,12 @@ def _l10n_ch_get_qr_vals(self, amount, currency, debtor_partner, free_communicat
             '0200',                                               # Version
             '1',                                                  # Coding Type
             acc_number,                                           # IBAN / QR-IBAN
-            'K',                                                  # Creditor Address Type
+            'S',                                                  # Creditor Address Type
             (self.acc_holder_name or self.partner_id.name)[:70],  # Creditor Name
-            creditor_addr_1,                                      # Creditor Address Line 1
-            creditor_addr_2,                                      # Creditor Address Line 2
-            '',                                                   # Creditor Postal Code (empty, since we're using combined addres elements)
-            '',                                                   # Creditor Town (empty, since we're using combined addres elements)
+            self.partner_id.street,                               # Creditor Address Line 1
+            self.partner_id.street2 or '',                        # Creditor Address Line 2
```

Comment:
No need to truncate to 70 here, as it was done previously?
```suggestion
            self.partner_id.street[:70],                          # Creditor Address Line 1
            (self.partner_id.street2 or '')[:70],                 # Creditor Address Line 2
```

Reply:
Actually it's even less:
- address line `StrtNmOrAdrLine1 ` is limited to 70 - OK
- but 2nd line `BldgNbOrAdrLine2` is for building number and is **limited to 16**

Other limitations:
- name is 70 (already enforced)
- town is limited to 35 (TODO)
- Postal code limited to 16 (not sure truncate is needed?)

Reference: https://www.six-group.com/dam/download/banking-services/standardization/qr-bill/ig-qr-bill-v2.3-en.pdf